backend/application/import_observations/parsers/prometheus_trivy/parser.py

```diff
@@ -0,0 +1,167 @@
+import json
+from typing import Optional
+
+import requests
+
+from application.core.models import Observation
+from application.core.types import Severity, Status
+from application.import_observations.models import Api_Configuration
+from application.import_observations.parsers.base_parser import (
+    BaseAPIParser,
+    BaseParser,
+)
+from application.import_observations.types import Parser_Type
+
+STATUS_MAPPING = {
+    "NOT_SET": "",
+    "EXPLOITABLE": Status.STATUS_OPEN,
+    "IN_TRIAGE": Status.STATUS_IN_REVIEW,
+    "RESOLVED": Status.STATUS_RESOLVED,
+    "FALSE_POSITIVE": Status.STATUS_FALSE_POSITIVE,
+    "NOT_AFFECTED": Status.STATUS_NOT_AFFECTED,
+}
+
+
+class PrometheusTrivy(BaseParser, BaseAPIParser):
```

Comment:
I would call it `Trivy Prometheus`, because Trivy is the scanner and should be in the front.

Reply:
fixed